jestsetup.js

```diff
@@ -0,0 +1,15 @@
+// Make Enzyme functions available in all test files without importing
+/* eslint-disable import/no-extraneous-dependencies */
+import Enzyme, { mount, render, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+global.shallow = shallow;
+global.render = render;
+global.mount = mount;
+
+window.matchMedia = () => ({ matches: true });
```

Comment:
This can be dropped. If I remember correctly, think this was added for date-time-picker mobile.